### DISCUSSION TITLE
identical values in json

### DISCUSSION THREAD

**Discussion by farmascript** [digitalfortress-tech/typeahead-standalone#84] [Q&A] [answered]
When you have identical values in 'title', typeahead gives only one result
```
/* Example Data source of Songs */
[
  { title: "God is Good", artist: "Don Moen" },
  { title: "God is Good", artist: "Paul Wilbur" },
  { title: "God is Good", artist: "Micheal Smith" },
  { title: "El Shaddai", artist: "Amy Grant" },
  ...
]
```
solution, according to the documentation:
identity(item) => `${item.title}${item.artist}`;

This does not work when i apply it. I suppose i am doing something wrong.
Where do i put it? In the source{}?
It should be ended with a comma, not with a semicolon?
I not only read about identity, but also identifier and identify.

I am confused.

Thank you for clarification.

**Comment by niketpathak** [comment, marked as answer]
Not reproducible for me. It works as expected for me. Kindly provide an example(codepen or jsfiddle) if this still does not work for you. Thanks!
